ansible/deploy.yml

```diff
@@ -1,4 +1,3 @@
 ---
-- include: python-setup.yml
```

Comment:
We don't need to install python on either 16 or 20. Both already come with the right python versions, 2 and 3 respectively. Our playbook works well with both. Installing python 2 on Ubuntu 20 breaks some other things, hence the removal.

Reply:
That's neat. Good catch.